docs/feature-gates.md

```diff
@@ -372,7 +367,7 @@ Refer to this [document](external-node.md) for more information.
 #### Requirements for this Feature
 
 Since Antrea Agent is running on an unmanaged VM/BM when this feature is enabled, features designed for K8s Pods are
-disabled. As of now, this feature requires that `AntreaProxy` and `NetworkPolicyStats` are also enabled.
+disabled. As of now, this feature requires that AntreaProxy and feature `NetworkPolicyStats` are also enabled.
```

Comment:
Why would ExternalNode require AntreaProxy, I think this was a typo, which meant to say AntreaPolicy? @wenyingd